src/Avalonia.Diagnostics/Diagnostics/ViewModels/MainViewModel.cs

```diff
@@ -258,9 +263,38 @@ public void RequestTreeNavigateTo(IControl control, bool isVisualTree)
         }
 
         public int? StartupScreenIndex { get; private set; } = default;
+        
+        [DependsOn(nameof(TreePageViewModel.SelectedNode))]
+        [DependsOn(nameof(Content))]
+        bool CanShot(object? parameter)
+        {
+            return Content is TreePageViewModel tree
+                && tree.SelectedNode != null
+                && tree.SelectedNode.Visual is VisualTree.IVisual visual
+                && visual.VisualRoot != null;
+        }
+
+        async void Shot(object? parameter)
+        {
+            if ((Content as TreePageViewModel)?.SelectedNode?.Visual is IControl control
+                && _screenshotHandler is { }
+                )
+            {
+                try
+                {
+                    await _screenshotHandler.Take(control);
```

Comment:
@maxkatz6 How do I play a sound for all supported platforms?

Reply:
Don't use file handler and use file picker handler